Add the Template file

superslalomracer opened the pull request:
This is for the purpose of internal use to help with formatting and editing of each other's code. 

Add markdown file to be used as a template for review and all
future notes. Open for edits until a date specified in the
related issue. After the file is final, all notes can be
edited to conform.

Template file describes convention for using example code as
both python and bash code, details an organization based on
main topics and a general commands at the end.

superslalomracer committed:
Add the Template file

Add markdown file to be used as a template for review and all
future notes. Open for edits until a date specified in the
related issue. After the file is final, all notes can be
edited to conform.

Template file describes convention for using example code as
both python and bash code, details an organization based on
main topics and a general commands at the end.

kevinaangstadt commented on Lecture_template.md:
````diff
@@ -0,0 +1,81 @@
+put two spaces for clean line breaks
+
+```
+python example code(foo)
````
GitHub md has a way of specifying the language for the block of code (e.g., ```python) 

Please update accordingly

kevinaangstadt commented on Lecture_template.md:
````diff
@@ -0,0 +1,81 @@
+
+#### Topic 3
+
+```
````
Same as Line 22, you can specify the language of this code block

kevinaangstadt commented on Lecture_template.md:
```diff
@@ -0,0 +1,81 @@
+# Lecture #: Topic   
+(mm/dd/2021)
+
+## **Overview**
```
Issue #33 discussion suggests removing this section

kevinaangstadt reviewed (changes requested):
Please address the comments in this review.

superslalomracer committed:
Update template file to match issues

Delete the 'Overview' section of the template.md file and update the
code blocks to be more consisten with markdown capabilities. Now,
every code block in the proposed file format specifies the language of
the code that is being demonstrated.

superslalomracer commented:
All requested changes have been made. New PR will be opened shortly.
